src/app/(cities)/gdansk/page.mdx

```diff
@@ -1,9 +1,58 @@
+import { env } from '../../../env';
+import { CityEmail } from '../../../components/CityEmail';
+import { Organizers } from '../../../components/Orginizers';
+import { FAQ } from '../../../components/FAQ';
+import { LocalGroups } from '../../../components/LocalGroups';
+import { CityBanner } from '../../../components/CityBanner';
+import { GdanskFAQ } from './_FAQ';
+import { GdanskOrganizers } from './_Organizers';
+import { EventSection } from '../../../components/EventSection';
+
 export const metadata = {
-	title: "meet.js Gdańsk",
+	metadataBase: new URL(`${env.SITE_URL}/gdansk`),
+	title: 'meet.js Gdańsk',
+	description: 'JavaScript meetups in Gdańsk',
+	keywords: 'meet.js, Gdańsk, JavaScript, meetup',
+	openGraph: {
+		title: 'meet.js Gdańsk',
+		description: 'JavaScript meetups in Gdańsk',
+		type: 'website',
+		locale: 'en_US',
+		url: `${env.SITE_URL}/gdansk`,
```

Comment:
`new URL(`${env.SITE_URL}/gdansk`)` as above maybe?

Reply:
<img width="424" alt="Screenshot 2025-04-18 at 12 07 03" src="https://github.com/user-attachments/assets/a1d076c5-7be0-4fe4-83dd-3d698f79529a" />

we do it in all pages, if you want you can refactor all pages